zrml/prediction-markets/src/lib.rs

```diff
@@ -116,10 +127,10 @@ mod pallet {
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        /// Allows the `DestroyOrigin` to immediately destroy a market.
+        /// Destroy a market, including its outcome assets.
```

Comment:
We're not 100% settled on what `destroy_market` should actually do, but the general idea is that it's a last resort and should only be used when a market creator has serious misbehaved (assassination markets come to mind). Therefore, we've first of all decided to slash the market creator's funds.

Other than that, yes, _all_ outcome tokens are slashed. We're not happy with this, because it means that everyone who participated in the market (traded/LPed) loses their outcome tokens. Now that I think about it, I'm not sure what happens with the ZTG of the pool. Probably stuck in the pool account forever. I'll have to look into that.